.github/bors.toml

```diff
@@ -5,9 +5,7 @@ timeout_sec = 1200
 status = [
   "style",
   "compile (stable)",
-  "compile (stable, pounder_v1_1)",
   "compile (1.59.0)",
-  "compile (1.59.0, pounder_v1_1)",
```

Comment:
Do we want CI to also run to check for Pounder v1.0 support as well?

Reply:
Sounds good. This was my thought as well, but didn't know what the plans might be fore v1.0 - are any of them still running in the wild?

Reply:
AFAIK a handful and few are used in practice.